src/transformers/integrations.py

```diff
@@ -269,74 +269,186 @@ def rewrite_logs(d):
     return new_d
 
 
-_is_deepspeed_zero3_enabled = None
+def is_true(config, key):
```

Comment:
I understand the short name, but would put this function private if we don't expand it a bit more to avoid potential conflicts.